README.md

```diff
@@ -1,3 +1,5 @@
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE.txt) [![CircleCI](https://img.shields.io/circleci/project/github/RedSparr0w/node-csgo-parser.svg)](https://circleci.com/gh/vintasoftware/django-react-boilerplate)
```

Comment:
@aericson @chocoelho I did see the other badge, but I added this alternative just to have something that 'matched' the other. I did not touch the original badge, I just wanted your opinion(s). I will remove it and make the other modifcations now.